pkg/apis/storage/types.go

```diff
@@ -46,6 +46,9 @@ type StorageClass struct {
 	// 512, with a cumulative max size of 256K
 	// +optional
 	Parameters map[string]string
+
+	// +optional
+	AllowVolumeExpand bool
 }
```

Comment:
We need to ensure that this flag is only accepted when the `ExpandVolume` feature flag is enabled. Users shouldn't be able to set this field if the feature flag is disabled.

Reply:
ok. i will add it in validation. thanks